pkg/resources/database.go

```diff
@@ -12,29 +12,29 @@ import (
 )
 
 var databaseSchema = map[string]*schema.Schema{
-	"name": &schema.Schema{
+	"name": {
 		Type:     schema.TypeString,
 		Required: true,
 		ForceNew: false,
 	},
-	"comment": &schema.Schema{
+	"comment": {
 		Type:     schema.TypeString,
 		Optional: true,
 		Default:  "",
 	},
-	"data_retention_time_in_days": &schema.Schema{
+	"data_retention_time_in_days": {
 		Type:     schema.TypeInt,
 		Optional: true,
 		Computed: true,
 	},
-	"from_share": &schema.Schema{
+	"from_share": {
```

Comment:
**[tfproviderlint]** <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub>
S006: schema of TypeMap should include Elem